packages/app-harness/package.json

```diff
@@ -30,30 +30,32 @@
         "watch": "tsc --watch --preserveWatchOutput --noEmit"
     },
     "dependencies": {
+        "@lightningjs/sdk": "^5.4.1",
+        "@react-native-community/cli": "^11.3.8",
```

Comment:
@mihaiblaga89  do we have recorded issue which is mitigated by enforcing this?
would like to avoid internal rn deps exposed in project as much as possible.